src/components/node-list/index.js

```diff
@@ -2,29 +2,31 @@ import React, { useEffect, useState } from 'react';
 import { connect } from 'react-redux';
 import utils from '@quantumblack/kedro-ui/lib/utils';
 import NodeList from './node-list';
-import { getFilteredItems, getGroups, getSections } from './node-list-items';
-import { toggleTagActive, toggleTagFilter } from '../../actions/tags';
 import {
-  toggleModularPipelineActive,
-  toggleModularPipelineFilter,
-} from '../../actions/modular-pipelines';
-import { toggleTypeDisabled } from '../../actions/node-type';
+  getFilteredItems,
+  getGroups,
+  isTagType,
+  isModularPipelineType,
+  isElementType,
+  isGroupType,
```

Comment:
I've crossed check these function imports with `node-list-items.js` and they are not declared in the file - are these functions and their relevant usage in this file still needed?